secboot/secboot_tpm.go

```diff
@@ -415,7 +415,7 @@ func buildPCRProtectionProfile(modelParams []*SealKeyModelParams) (*sb_tpm2.PCRP
 			PCRAlgorithm:  tpm2.HashAlgorithmSHA256,
 			LoadSequences: loadSequences,
 			// TODO:UC20: set SignatureDbUpdateKeystore to support applying forbidden
-			//            signature updates to blacklist signing keys (after rotating them).
+			//            signature updates to excluded signing keys (after rotating them).
```

Comment:
There's a little change of meaning here. To maintain the old meaning, we should use "exclude" here. Or was the old text wrong?

Reply:
You are right, I think 'exclude' makes more sense?